OpenRA.Mods.Common/Activities/Air/HeliReturnToBase.cs

```diff
@@ -44,13 +44,28 @@ public override Activity Tick(Actor self)
 			{
 				var rearmBuildings = heli.Info.RearmBuildings;
 				var nearestHpad = self.World.ActorsHavingTrait<Reservable>()
-									.Where(a => a.Owner == self.Owner && rearmBuildings.Contains(a.Info.Name))
-									.ClosestTo(self);
+					.Where(a => a.Owner == self.Owner && rearmBuildings.Contains(a.Info.Name))
+					.ClosestTo(self);
```

Comment:
Eventually we might to add the behaviour the harvesters use for refineries now (ie, three harvs per refinery, if it's full, move on to the next one).